lib/rom/factory/tuple_evaluator.rb

```diff
@@ -80,7 +80,10 @@ def evaluate(*traits, **attrs)
       # @api private
       def evaluate_values(attrs)
         attributes.values.tsort.each_with_object({}) do |attr, h|
-          deps = attr.dependency_names.map { |k| h[k] }.compact
+          deps = attr.dependency_names.map do |key|
+             h[remove_underscore(key)]
```

Comment:
this is super minor but I wouldn't reveal the details in the name of the method, that is something like `dependency_name` suits better. Next time you change the rules you'll need to change only the body of the method

Reply:
Totally agree with it @flash-gordon 👍